unityfiles/Assets/Scripts/Helper/Util/WUArrays.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading;
+using System.Linq;
+
+using UnityEngine;
+
+
+static class WUArrays
```

Comment:
Hva er WU?

Reply:
som i WUArrays?

Reply:
Woldset Utillety det generiske prefiken jeg bruker på mine utillety fungsjoner i alle forsjellige programeringspråk

burde kansje bytte

Reply:
yes please

Reply:
Denne er kopiert fra en av mine private repo (ett div utillity stuff) hvis det ikke er så alt for mye trøbbel fåretrekker jeg og beholde navnet